### PR TITLE
[NG] Add docker_orchestrator doc

### PR DESCRIPTION
**Pull request by mbounabi**

### CODE REVIEW

File: artemis/docker_orchestrator.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    `CONFIG_FILE=<my_config_file> docker_orchestrator.py test <coverage_1> <coverage_2>`
+
+The orchestrator will find the coverages to run in the [instances list](https://github.com/CanalTP/navitia-docker-compose/blob/master/artemis/artemis_custom_instances_list.yml) from the navitia-docker-compose project.
+The name of the coverages to be passed in the command above is the key from the instances list.
+_ex: airport-01, fr-auv, etc..._
+
+To pull the latest docker images from [Dockerhub](https://hub.docker.com/u/navitia/) before running the tests, the parameter `-p | --pull` can be added to the test command.
+
+The orchestrator also allows to store logs from the tests session. To do so, use the `-l | --logs` parameter with the test command. The logs will be stored in a `logs` folder created in the `RESPONSE_FILE_PATH` folder (by default: <artemis_root>/output).
+Here, a folder is created for each coverage run with the XML file from the pytest results and the logs from every containers run.
```
Comment (kadhikari) [inline artemis/docker_orchestrator.md:47]: .. from every containers run. -> .. from every container run.

File: artemis/readme.md [hunk 2]
```diff
@@ -46,6 +46,11 @@ Once all containers are running, it's time to run the tests.
     - Of course, every pytest invocation arguments can also be used on this command.\
       *Ex: if you want to run a specific test (or a tests class for a coverage)*`-k <tests_to_run>`
 
+### Or... Docker orchestrator
```
Comment (kadhikari) [inline artemis/readme.md:49]: Or... ??

File: artemis/docker_orchestrator.md (hunk 1 above)
Comment (mbounabi) [reply artemis/docker_orchestrator.md:47]: Not sure about this...

File: artemis/readme.md (hunk 2 above)
Comment (mbounabi) [reply artemis/readme.md:49]: Yes. You can run the 2 steps above manually OR run docker_orchestrator

File: artemis/readme.md [hunk 3]
```diff
@@ -46,6 +46,11 @@ Once all containers are running, it's time to run the tests.
...
+
+See [here](https://github.com/CanalTP/artemis/blob/master/artemis/docker_orchestrator.md) how the orchestrator can handle the 2 setps above.
```
[the remark this answers is not in the document]
Comment (mbounabi) [reply artemis/readme.md:51]: In the next PR, due to mergify...

**Pull request merged**